uiowa/uiowa
# Hotfix HR media

Related to the helpdesk ticket.

Fix embedded videos on HR. Might have to be expanded to address uids_base v2 sites.

## To Test

- pull and sync HR
- `blt frontend` (uiowa_bootstrap and hr both need to build)
- Go to https://hr.local.drupal.uiowa.edu/advanced-layout-guide-not-publication and look at the videos section. It should fix the broken videos. Compare to PROD (unpublished) where two videos (small ones (default/full)) aren't rendering.
<img width="308" alt="Screen Shot 2021-02-18 at 11 59 02 AM" src="https://user-images.githubusercontent.com/4663676/108400688-21962380-71e1-11eb-8b7d-d816dad37ee1.png">
